services/wfm/src/main/java/org/openkilda/wfm/topology/floodlightrouter/bolts/DiscoveryBolt.java

```diff
@@ -145,11 +145,11 @@ public void prepare(Map map, TopologyContext topologyContext, OutputCollector ou
 
 
     @Override
-    public void send(Message message, String outputStream) {
+    public void send(Message message, String outputStream, boolean lookupKey) {
```

Comment:
Looks like lookupKey param now always false

Reply:
since it's public it's OK, need to revisit this while fixing kafka keys, IMHO